### ISSUE TITLE
[brotli] Build error

### ISSUE THREAD

**Issue by TheVeryDarkness** [microsoft/vcpkg#34553]
Package: brotli:wasm32-emscripten -> 1.1.0

**Host Environment**

- Host: arm64-osx
- Compiler: Clang 18.0.0
-    vcpkg-tool version: 2023-09-15-2023.09.15
    vcpkg-scripts version: b40de4489 2023-10-11 (7 days ago)

**To Reproduce**

`vcpkg install harfbuzz:wasm32-emscripten`
**Failure logs**

```
-- Using cached google-brotli-v1.1.0.tar.gz.
-- Cleaning sources at .../vcpkg/buildtrees/brotli/src/v1.1.0-3d84596e32.clean. Use --editable to skip cleaning for the packages you specify.
-- Extracting source .../vcpkg/downloads/google-brotli-v1.1.0.tar.gz
-- Applying patch install.patch
-- Applying patch fix-arm-uwp.patch
-- Applying patch pkgconfig.patch
-- Using source at .../vcpkg/buildtrees/brotli/src/v1.1.0-3d84596e32.clean
-- Found external ninja('1.11.1').
-- Configuring wasm32-emscripten
-- Building wasm32-emscripten-dbg
-- Building wasm32-emscripten-rel
-- Fixing pkgconfig file: .../vcpkg/packages/brotli_wasm32-emscripten/lib/pkgconfig/libbrotlicommon.pc
-- Fixing pkgconfig file: .../vcpkg/packages/brotli_wasm32-emscripten/lib/pkgconfig/libbrotlidec.pc
-- Fixing pkgconfig file: .../vcpkg/packages/brotli_wasm32-emscripten/lib/pkgconfig/libbrotlienc.pc
-- Fixing pkgconfig file: .../vcpkg/packages/brotli_wasm32-emscripten/debug/lib/pkgconfig/libbrotlicommon.pc
-- Fixing pkgconfig file: .../vcpkg/packages/brotli_wasm32-emscripten/debug/lib/pkgconfig/libbrotlidec.pc
-- Fixing pkgconfig file: .../vcpkg/packages/brotli_wasm32-emscripten/debug/lib/pkgconfig/libbrotlienc.pc
CMake Error at scripts/cmake/vcpkg_copy_tools.cmake:36 (message):
  Couldn't find tool "brotli":

      ".../vcpkg/packages/brotli_wasm32-emscripten/tools/brotli/brotli" does not exist
Call Stack (most recent call first):
  ports/brotli/portfile.cmake:26 (vcpkg_copy_tools)
  scripts/ports.cmake:168 (include)



```

**Comment by TheVeryDarkness** [comment]
There exists a brotli.js, but I think it should not be what we want.

**Comment by TheVeryDarkness** [comment]
It seems this has been fixed in brotli 1.1.0#1
